jekyll/_cci2/github-trigger-event-options.adoc

```diff
@@ -14,21 +14,37 @@ When you create a trigger for a GitHub App pipeline, you can choose from a set o
 
 == Introduction
 
-When you set up a project in CircleCI you will also set up your first pipeline and trigger. This first trigger is, by default, an "all push" trigger, which corresponds to the link:https://docs.github.com/en/webhooks/webhook-events-and-payloads#push[GitHub App push event]. If you have a GitHub App account with CircleCI, you will see trigger options when creating a project in the CircleCI web app, and you have the option to change the trigger options for your first trigger. You can also set up additional triggers during project creation, or setup/edit later in your menu:Project settings[Triggers].
+CircleCI projects have pipelines and trigger.
 
-.Trigger options during project creation
-image::triggers/create-project-run-on.png[Create a project trigger and access trigger options]
+* Pipelines define the various CI/CD 'units' of work that make up your project.
+* Triggers define when and how to initiate a pipeline.
+
+Configure triggers under menu:Project settings[Triggers]. You can configure triggers to initiate pipelines on a variety of events
```

Comment:
> You can configure triggers to initiate pipelines on a variety of events
> * Non-push events, for example, a pull request being opened
> * Push events, for example, only pushes to a default branch, or pushes to a tag.

This makes it sound like "push events" and "non push events" are the only options, I think we want to say "a variety of Git events". 
Or maybe something like "you can configure triggers to initiate pipelines on a variety of events. This page focuses on events from Github, but you can find the full list of events that can trigger a pipeline [here](https://circleci.com/docs/triggers-overview)